docs/warehouse/developing_dbt_models.md

```diff
@@ -182,26 +200,35 @@ The first step is running your changes in the test/staging environment. You can
 What to test/check will vary based on what you're doing, but below are some example things to consider.
 
 #### Column values
+
 Are the values in your column/model what you expect? For example, are there nulls? Does the column have all the values you anticipated (for example, if you have a day of the week column, is data from all 7 days present)? If it's numeric, what are the minimum and maximum values; do they make sense (for example, if you have a percentage column, is it always between 0 and 100)? What is the most common value?
+
 * To check nulls:
-    ```
+
+    ```{sql}
```

Comment:
Looks like JupyterBook markdown doesn't know about SQL, can you revert these? I don't [see an equivalent here](https://jupyterbook.org/en/stable/content/content-blocks.html)
![image](https://github.com/cal-itp/data-infra/assets/55149902/1f615f21-8200-4fee-85d3-4ffb1bb129d3)

Reply:
That was just a labeling issue; I shouldn't have fenced them in brackets. Fixing in a new commit momentarily.

Reply:
Actually it looks like it might be `sql` without curly braces

Reply:
See https://raw.githubusercontent.com/cal-itp/data-infra/main/docs/analytics_tools/jupyterhub.md for reference

Reply:
Uses `python` and `bash`

Reply:
Ope sorry, my GH hadn't refreshed to show your response

Reply:
Resolved in [a6ac2ef](https://github.com/cal-itp/data-infra/pull/2896/commits/a6ac2ef85367618c81f49b0a45f13811ff4043b0)